src/ts/uimanager.ts

```diff
@@ -247,7 +260,7 @@ export class UIManager {
       let adRequiresUi = false;
       if (isAd) {
         let ad = adStartedEvent.ad;
-        // for now only linear ads can request an UI
+        // for now only linear ads can request a UI
```

Comment:
Shouldn't it be `an`

Reply:
https://www.quora.com/Is-it-an-user-or-a-user